contrib/oracles.asciidoc

```diff
@@ -45,18 +45,18 @@ The primary functions of an oracle may therefore be summarized as follows:
+Authenticity proofs are cryptographic guarantees that prove that data has not been tampered with. Based on a variety of attestation techniques (e.g. digitally signed proofs), they effectively shift the trust from the data carrier to the attestor, i.e. the provider of the attestation method. By verifying the authenticity proof on-chain, smart contracts are able to verify the integrity of the data before operating upon it. An authenticity proof that has been implemented by Oraclize [1] and is currently available for data queries from the Ethereum main network is the TLSNotary Proof [2]. TLSNotary Proofs allow a client to provide evidence to a third party that HTTPS web traffic occurred between the client and a server; while HTTPS is itself secure, it doesn't support data signing, so TLSNotary (via PageSigner [3]) signatures are used. TLSNotary Proofs leverage the Transport Layer Security (TLS) protocol, enabling the TLS master key, which signs the data after it has been accessed, to be split between three parties: the server (the oracle), an auditee (Oraclize), and an auditor. As an auditor, Oraclize use an Amazon Web Services (AWS) virtual machine instance that can be verified as unmodified since instantiation [4]. This AWS instance stores the TLSNotary secret, allowing it to provide honesty proofs. While offering higher assurances against data tampering than a purely trusted query/response mechanism, this approach assumes that Amazon itself will not tamper with the VM instance.
```

Comment:
> ... the server (the oracle), an auditee (Oraclize), and an auditor. As an auditor, Oraclize use an Amazon Web Services

This is confusing. In the first sentence, Oraclize is mentioned as the auditee. In the second sentence, it is referred to as an auditor. Also, grammar correction: "Oraclize use**s**"